study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/403.jsp

```diff
@@ -1,10 +1,148 @@
-<div class="txt-center pt-xlg pb-xlg">
-  <div class="ErrorBox">
-    <img src="/studybuilder/images/404.png" alt="">
-    <div class="bigTxt">Oops 403!</div>
-    <div class="subTxt">Access is denied <br> You do not have permission to access this page!</div>
-  </div>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
```

Comment:
Should this have the copyright header?

Reply:
In the existing JSP files we have not added copyright header